.github/no-response.yml

```diff
@@ -0,0 +1,13 @@
+# Configuration for probot-no-response - https://github.com/probot/no-response
+
+# Number of days of inactivity before an Issue is closed for lack of response
+daysUntilClose: 14
+# Label requiring a response
+responseRequiredLabel: needs response
```

Comment:
The existing needs response bot should be shuttered so they're not competing.

Reply:
Aye, will do that before this is merged but thanks for the reminder 🙇

Reply:
Done!